torch/_dynamo/types.py

```diff
@@ -13,6 +13,7 @@
     Union,
 )
 
+import torch
```

Comment:
??

Reply:
was getting mypy violations in lint on L36